frontend/javascripts/oxalis/view/novel_user_experiences/welcome_toast.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, { useEffect } from "react";
+import { Button, notification } from "antd";
+import { useSelector } from "react-redux";
+import features from "features";
+
+function showWelcomeToast() {
+  notification.open({
+    message: (
+      <div>
+        <span className="logo" />
+        welcome to webKnossos!
+      </div>
+    ),
+    description: (
+      <div>
+        You are viewing this dataset on webKnossos. webKnossos is an online tool to visualise and
+        collaboratively annotate large image datasets. Try out the annotation features on the public
+        dataset below. If this is for you, sign up for free, upload your data and get started!
+        <div style={{ marginTop: 12, marginLeft: 200 }}>
+          <Button
+            type="primary"
+            href="/auth/login"
+            target="_blank"
+            rel="noopener noreferrer"
+            style={{ background: "#256ff7" }}
+          >
+            Sign Up
+          </Button>
+          <Button
+            type="default"
+            href="https://webknossos.org/features"
+            target="_blank"
+            rel="noopener noreferrer"
+            style={{ borderColor: "#eaeaea", marginLeft: 12 }}
+          >
+            Learn More
+          </Button>
+        </div>
+      </div>
+    ),
+    className: "webknossos-welcome-toast",
+
+    style: {
+      width: 600,
+    },
+    duration: 0,
+  });
+}
+
+export default function WelcomeToast() {
+  const activeUser = useSelector(state => state.activeUser);
+
+  useEffect(() => {
+    // if (features().isDemoInstance) {
+    //   return;
+    // }
```

Comment:
comment in before merging

Reply:
Is this condition correct or does it need to be negated? I would've thought this to be especially relevant for the demo instance.

Reply:
you're right, this needs to be inverted!

Reply:
good catch!